charts/aws-fluent-bit/README.md

```diff
@@ -7,12 +7,49 @@ A helm chart intended to be used with the [AWS marketplace version of Calyptia F
 Ensure a service account is set up with the `arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage` policy to register usage correctly.
 
 ```shell
-eksctl create cluster
+eksctl create cluster --name <clustername> --version 1.2x --region <region-code>
+```
+
+By default, EKS has an OpenID Connect(OIDC) issuer URL to provide federated access. AWS IAM supports IDP's that are compatible with OpenIDConnect to establish trust relationships to access the AWS account.
+
 
+### Adding the [IdP](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) with the AWS IAM service
+
+```
 eksctl utils associate-iam-oidc-provider --cluster=<clusterName> --approve
+```
+
+You could check if the IdP has been added by going to the AWS IAM console in Identity providers under Access management
+
+### Creating the IAM role and the Kubernetes Service Account
+The below command will create the role and the kubernetes service account in the respective namespace. 
 
-eksctl create iamserviceaccount --cluster=<clusterName> --name=eks-calyptia-fluentbit --namespace=<serviceAccountNamespace> --attach-policy-arn=arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage --approve
 ```
+eksctl create iamserviceaccount \
+ --name irsa-calyptia-fluentbit \
+ --namespace calyptia-fluentbit \
+ --cluster <clusterName> \
+ --role-name "<enterthenameoftheroletocreate>" \
+ --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage \
+ --approve \
+ --override-existing-serviceaccounts
+```
+
+verify if the trust relationship is configured correctly by running the below command:
```

Comment:
What should the output show so we can verify?
Same for the following - either link out to a docs URL or show a sample output that is correct.